v0/destinations/vero/data/VeroTagsConfig.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "sourceKeys": ["userId", "context.traits.email", "traits.email", "anonymousId"],
+    "destKey": "id"
+  },
+  {
+    "sourceKeys": "integrations.Vero.tags.add",
```

Comment:
We should consider `destinationCanonicalNames` when mapping from integrations object. If required we can add support using metadata field

Reply:
Done